Source/OCMockTests/OCMockObjectPartialMocksTests.m

```diff
@@ -60,6 +60,43 @@ - (void)bar:(id)someArgument // maybe we should make it explicit that the arg is
 
 @end
 
+@interface TestClassThatObservesFoo : NSObject
+{
+  id observedObject;
+}
+@end
+
+@implementation TestClassThatObservesFoo
+
+- (instancetype)initWithObject:(id)object
+{
+  if((self = [super init]))
+  {
+    observedObject = object;
+  }
+  return self;
+}
+
+- (void)startObserving
+{
+  [observedObject addObserver:self forKeyPath:@"foo" options:0 context:NULL];
+}
+
+- (void)stopObserving
+{
+  if(observedObject)
+  {
+    [observedObject addObserver:self forKeyPath:@"foo" options:0 context:NULL];
```

Comment:
I guess this should be `removeObserver:`. If I make that change then the test fails on line 706. I can't reproduce the problem with the autorelease pool mentioned there.